src/Microsoft.DotNet.Darc/src/Darc/Operations/GetDependencyFlowGraphOperation.cs

```diff
@@ -38,7 +38,7 @@ public override async Task<int> ExecuteAsync()
                 defaultChannels.Add(
                     new DefaultChannel(0, "https://github.com/dotnet/arcade", true)
                     {
-                        Branch = "refs/heads/master",
+                        Branch = "master",
```

Comment:
Should LN 48 also have the `refs/heads/` removed?

Reply:
good catch.  that block was added after this PR was created.

Reply:
Yeah that would do it.